tutorial/source/index.rst

```diff
@@ -56,6 +56,7 @@ Welcome to Pyro Examples and Tutorials!
    :maxdepth: 2
    :caption: Code Examples:
 
+   csj
```

Comment:
Where is `tutorial/source/cjs.rst`?

Reply:
It looks like you have completed step 2 before step 1. I'll merge now, but could you please add this file in a follow-up PR?